libs/utils/src/scripts/generate_env_file.ts

```diff
@@ -0,0 +1,102 @@
+/* istanbul ignore file */
+import * as readline from 'readline';
+import * as fs from 'fs';
+import { join } from 'path';
+import yargs from 'yargs';
+import { EnvironmentFlagName, getFlagDetails } from '../environment_flag';
+
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+interface GenerateEnvFileArguments {
+  outputPath?: string;
+  isVxDev?: boolean;
+  help?: boolean;
+}
+
+const flagOptions: Map<string, boolean> = new Map();
+function question(flagName: string, enable: boolean) {
+  return new Promise<void>((resolve) => {
+    rl.question(
+      `Enable ${flagName}? (${enable ? 'Y/n' : 'y/N'}):`,
```

Comment:
Should we suggest a default value here and allow just using the default?

Reply:
There is a default, so it will either tell you to enter Y/n (which defaults to yes) or y/N (which defaults to no)

Reply:
Ah ok I didn't understand that's what the different capitalization meant